### ISSUE TITLE
Auto-Return to starting position after zooming in/out from selected object. 

### ISSUE THREAD

**Issue by MADAMastronomer** [Stellarium/stellarium#2867]
Mary from the Adelaide Planetarium here.
Is there an auto-return feature in Stellarium?  Currently have to do it manually and it looks awful on the fulldome when we have to do this during our sessions. Would love an auto-return feature/button. Martin Lewicki and I looked for a config.ini file on our MAC but we couldn't find it. Can't find anything to help me in the huge userguide. 
We use an iMac (newest one, M-1 chip) for fulldome software.
Thank you.
 

### Actual Behaviour
Describe or maybe attach a screenshot?

### System
* Stellarium version: <Name of downloaded installable file?>
* Operating system: <Name, version number>
* Graphics Card: <Manufacturer (likely Intel, NVidia, AMD?), Model (HD, Geforce, Radeon..., with model number), driver version?>
* Screen type (if applicable): Resolution, HighDPI, scaling

### Logfile 
If possible, attach the logfile `log.txt` from your user data directory. Look into the Guide for its location.

**Comment by alex-w** [comment]
Please describe typical session where you need auto-return feature and what is "auto-return feature" for you?

P.S. Did you checked auto-return settings in F2/Tools?

**Comment by MADAMastronomer** [comment]
Yes I have looked through and can’t find what I need to help us. Martin and I have both looked and we’ve spent hours trying to figure out how.  It may help explain what I’m after if I run through what we do in our sessions here:

When using stellarium on the Planetarium’s fulldome, I have it set up so South is at the bottom of the circle (horizon) on my iMac and North is at the top.  I speed up time and show the constellations and how they move as the earth rotates. There are objects I like to zoom into and show up close such as M31,M42 and M45. When I select and centre the object, eg M31 , I zoom in and show it up close, then  press \ to zoom out.  When I zoom back out I have to manually rotate it back (using mouse or directional keys)  so that the cardinal points and the ecliptic are back in the correct starting place (as seen from Adelaide).  Having to manually move it back each time is time consuming and frustrating. This is what I meant by Auto-return. A means of getting the sky back to its original orientation (before the zoom in) smoothly?
Hope this makes sense.

Kind Regards

Mary Adam
Astronomy Educator/Lecturer
Adelaide Planetarium
University Of South Australia
Mawson Lakes
GPO Box 2471, Adelaide SA 5001
PH: +61 411 519 468
email: ***@***.******@***.***>
web: www.unisa.edu.au/planetarium/<http://www.unisa.edu.au/planetarium/>

[Planetarium_50 years_email signature]

[Plntrum_12_01]

***@***.***
UniSA acknowledges that we respect the Kaurna, Barngarla and Boandik peoples spiritual relationship with their country.  We recognise and respect their cultural heritage, beliefs and relationship with the land. We acknowledge that they are of continuing importance to those people living today. We acknowledge the diversity of Aboriginal peoples, past and present

**Comment by alex-w** [comment]
Please try enable "Auto-direction at zoom out" option via F4/Tools dialog

**Comment by alex-w** [comment]
@MADAMastronomer any news, please

**Comment by MADAMastronomer** [comment]
Hi, I haven’t been at work today, but I tried again last night and now the iMac won’t stay centred when I select that auto option under tools.
I think it may be an issue with our computer.
I will try again tomorrow at work.
Ps. On the PC it works fine, no issues at all.

Kind Regards,

Mary Adam
Astronomer/Astronomy Educator
Adelaide Planetarium
University Of South Australia
Mawson Lakes
GPO Box 2471, Adelaide SA 5001<x-apple-data-detectors://1/0>
PH: +61 411 519 468<tel:+61%20411%20519%20468>
         08 8302 3138<tel:08%C2%A08302%C2%A03138>
email: ***@***.******@***.***>
web: www.unisa.edu.au/planetarium/<http://www.unisa.edu.au/planetarium/><http://www.unisa.edu.au/planetarium/>





UniSA acknowledges that we respect the Kaurna, Barngarla and Boandik peoples spiritual relationship with their country.  We recognise and respect their cultural heritage, beliefs and relationship with the land. We acknowledge that they are of continuing importance to those people living today. We acknowledge the diversity of Aboriginal peoples, past and present.

On 24 Nov 2022, at 7:37 pm, Alexander V. Wolf ***@***.***> wrote:

﻿

@MADAMastronomer<https://github.com/MADAMastronomer> any news, please

**Comment by alex-w** [comment]
@MADAMastronomer any news and please check release 1.2

**Comment by MADAMastronomer** [comment]
Thank you Alexander, got it working now.

I did want to ask you if it's possible to have the image of Sagittarius A* added to the centre of the galaxy? In our Planetarium we do zoom in to the centre. Martin said something about adding patches and I've read about it, but it's beyond my understanding.

Kind Regards,

Mary Adam
Astronomer/Astronomy Educator
School of STEM
University Of South Australia
T +61 8 8302 3138
M +61 0411 519 468
***@***.******@***.***>





***@***.***



***@***.***

***@***.***


UniSA acknowledges that we respect the Kaurna, Barngarla and Boandik peoples spiritual relationship with their country.  We recognise and respect their cultural heritage, beliefs and relationship with the land. We acknowledge that they are of continuing importance to those people living today. We acknowledge the diversity of Aboriginal peoples, past and present.

**Comment by gzotti** [comment]
If you mean the "donut" image, this was created in the radio spectrum, right? So it is not visual and actually contradicts the visual paradigm of the program. Adding this in a script is however quite easy. See http://stellarium.org/doc/1.x/classStelMainScriptAPI.html#a7f4d145e871a3b5f9b24a3ace11198d3 

The core object has 4 similar functions to insert an image onto a location specified by coordinates.

**Comment by MADAMastronomer** [comment]
Thank you. I’ll give it a shot.

Kind Regards,

Mary Adam
Astronomer/Astronomy Educator
Adelaide Planetarium
University Of South Australia
Mawson Lakes
GPO Box 2471, Adelaide SA 5001<x-apple-data-detectors://1/0>
PH: +61 411 519 468<tel:+61%20411%20519%20468>
         08 8302 3138<tel:08%C2%A08302%C2%A03138>
email: ***@***.******@***.***>
web: www.unisa.edu.au/planetarium/<http://www.unisa.edu.au/planetarium/><http://www.unisa.edu.au/planetarium/>





UniSA acknowledges that we respect the Kaurna, Barngarla and Boandik peoples spiritual relationship with their country.  We recognise and respect their cultural heritage, beliefs and relationship with the land. We acknowledge that they are of continuing importance to those people living today. We acknowledge the diversity of Aboriginal peoples, past and present.

On 11 Jan 2023, at 8:15 pm, Georg Zotti ***@***.***> wrote:

﻿

If you mean the "donut" image, this was created in the radio spectrum, right? So it is not visual and actually contradicts the visual paradigm of the program. Adding this in a script is however quite easy. See http://stellarium.org/doc/1.x/classStelMainScriptAPI.html#a7f4d145e871a3b5f9b24a3ace11198d3<http://stellarium.org/doc/1.x/classStelMainScriptAPI.html#a7f4d145e871a3b5f9b24a3ace11198d3>

The core object has 4 similar functions to insert an image onto a location specified by coordinates.
